builtin/logical/aws/path_static_roles.go

```diff
@@ -202,14 +205,38 @@ func (b *backend) pathStaticRolesDelete(ctx context.Context, req *logical.Reques
 }
 
 func (b *backend) validateRoleName(name string) error {
+	if name == "" {
+		return errors.New("empty role name attribute given")
+	}
 	return nil
 }
 
 func (b *backend) validateIAMUserExists(ctx context.Context, req *logical.Request, username string) error {
+	c, err := b.clientIAM(ctx, req.Storage)
+	if err != nil {
+		return fmt.Errorf("unable to validate username '%s': %w", username, err)
```

Comment:
Similar to above, perhaps this could use `%q` for automatic quotes instead of `'%s'`?